test/release/examples/users-guide/base/forloops.chpl

```diff
@@ -0,0 +1,40 @@
+config param tryIllegal = true;
+
+for i in 1..5 do
+  writeln("hello #", i);
+writeln();
+
+var i: int;
+for i in 1..5 do
+  writeln(i);
+writeln("i is: ", i);
+writeln();
+
+if tryIllegal then
+for i in 1..5 do
+  i += 1;        // Illegal: i is 'const' for ranges
```

Comment:
It hasn't in the sections I've written, but I'm imagining that the section that introduces it will come before this one.